### PR TITLE
MS teams - adding imports for permissions use

### PR DESCRIPTION
**Pull request by ohaim1008**
## Status
In Progress

## Related Issues
related: https://github.com/demisto/etc/issues/31658

## Description
Adding imports in order to get access to allow a user to create a meeting in MS Teams.

### CODE REVIEW

**Comment by glicht** [comment]
For which integration is this?

**Comment by ohaim1008** [comment]
@glicht MS Teams, I'm adding a command to it.

**Comment by jochman** [comment]
I'm suggesting that you'll put the import in the integration itself as it is much clearer what commands you're using.

**Comment by glicht** [comment]
I might be missing something. Isn't ms teams in python?

**Comment by Itay4** [comment]
@ohaim1008 not sure what you're trying to do here.. teams is in python and uses the `demisto/teams` docker image

**Comment by ohaim1008** [comment]
@glicht @Itay4 It is, but in order to create a meeting Teams is using Skype for business, and we need to create an application access policy using ps commands - https://docs.microsoft.com/en-us/graph/cloud-communication-online-meeting-application-access-policy

**Comment by ohaim1008** [comment]
We will run those command in a separate script with a powershell docker image.

**Comment by jochman** [comment]
@ohaim1008 you might want to create a new docker image based on this one with MS-Teams module

File: docker/pwsh-exchange/Dockerfile [hunk 1]
```diff
@@ -3,4 +3,6 @@ FROM demisto/powershell-ubuntu:7.1.1.16151
 
 RUN pwsh -c "Set-PSRepository -Name PSGallery -InstallationPolicy Trusted -ErrorAction Stop"
 RUN pwsh -c "Install-Module -ErrorAction Stop -Scope AllUsers -Name ExchangeOnlineManagement -RequiredVersion 2.0.4-Preview4 -AllowPrerelease -Force"
-RUN pwsh -c "Install-Module -Name SelfSignedCertificate -AllowPrerelease -Force"
\ No newline at end of file
+RUN pwsh -c "Install-Module -Name SelfSignedCertificate -AllowPrerelease -Force"
+RUN pwsh -c "Install-Module MicrosoftTeams -Force"
+RUN pwsh -c "Import-Module MicrosoftTeams -Force"
```
Comment (jochman) [inline docker/pwsh-exchange/Dockerfile:8]: You should put this `Import-Module` in the integration itself as the programmer should be aware of the import.

**Comment by ohaim1008** [comment]
@jochman I will open a new docker image for it.

**Comment by glicht** [comment]
@Itay4 @ohaim1008 please check first that there isn't a python based solution

**Comment by ohaim1008** [comment]
@glicht @Itay4 I'll explain the issue better:
We must set up permissions to a users in order to allow them to create a meeting.
We were manage to do it locally with those ps commands and wanted to release a script that will ease the process for the customer, I'll mention that the customer will have to put its admin credentials as an input to the script.
We thought about the option to just document it in the readme but thought that a script will be better.
wdut?

@michalgold @evisochek FYI

**Pull request closed without merging**